src/components/Select/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { PureComponent } from 'react'
+import { string, bool, array, func, object } from 'prop-types'
+import Select, { components } from 'react-select'
+
+export default class SelectInput extends PureComponent {
+  static propTypes = {
+    onChange: func.isRequired,
+    onFocus: func,
+    options: array,
+    placeholder: string,
+    isSearchable: bool,
+    defaultValue: object,
+    components: object,
+  }
+
+  static defaultProps = {
+    options: [],
+    placeholder: 'Select one',
+    isSearchable: false,
+  }
+
+  state = { isOpen: false }
+
+  handleOpenMenu = () =>
+    this.setState({ isOpen: true })
+
+  handleCloseMenu = () =>
+    this.setState({ isOpen: false })
+
+  renderDropdownIndicator = () => {
+    const { isOpen } = this.state
+    const DropdownIndicator = props =>
```

Comment:
Should this be defined outside of the render method?  It doesn't need to be redefined on every render, right?